pkg/asset/installconfig/azure/session.go

```diff
@@ -117,18 +133,47 @@ func getCredentials(fs auth.FileSettings) (*Credentials, error) {
 	if tenantID == "" {
 		return nil, errors.New("could not retrieve tenantId from auth file")
 	}
+	activeDirectoryEndpoint := fs.Values[auth.ActiveDirectoryEndpoint]
```

Comment:
Having Active Directory authentication is fine, but we should keep the default authentication behavior and not assume or fail if there is no Active Directory endpoint.

Reply:
This isn't for authentication but calculating the endpoints required for azurerm. https://docs.microsoft.com/en-us/azure/azure-government/documentation-government-developer-guide#endpoint-mapping